tests/apichecks/testdata/exceptions/missingrefs.txt

```diff
@@ -6,6 +6,7 @@
 [refs] crd=alloydbbackups.alloydb.cnrm.cloud.google.com version=v1beta1: field ".spec.encryptionConfig.kmsKeyName" should be a reference
 [refs] crd=apigatewaygateways.apigateway.cnrm.cloud.google.com version=v1alpha1: field ".spec.apiConfig" should be a reference
 [refs] crd=apigeenataddresses.apigee.cnrm.cloud.google.com version=v1alpha1: field ".spec.instanceId" should be a reference
+[refs] crd=backupdrbackupplans.backupdr.cnrm.cloud.google.com version=v1alpha1: field ".spec.backupVault" should be a reference
```

Comment:
Please fix this field to use a reference.

Reply:
I'm working on backupVault, but our current tooling conductor doesn't handle multiple resources under the same API group very well. I'd prefer to merge this first, land backupVault, and then address the reference issue (I've added a TODO in the code).

Reply:
kk. works for me. Another option is that you can run generate-types to create the backupvault_reference.go file and comment out other code you don't want to checkin.

Reply:
Addressing in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/4017